global.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Even more than when the next OS image is updated and doesn't have this one it would break.

Reply:
Well you are missing the fact that the solution does not require NET 8 SDK because nothing targets net8.0 yet. This will lead to the fact that depending the SDK you have (say 6 or 8) you will get different output folders (UseArtifactsOutput) and the tests will break.

So when the targets will be updated to net8.0 it will implicitly require the SDK and this file can be removed. But I discovered that there's regex changes that break the tests when run under net8.0 so requires some work.

Reply:
So this is to enfore that 8.0 sdk is used because of `UseArtifactsOutput`, ok.